cms/server/contest/templates/contest.html

```diff
@@ -161,11 +165,11 @@ <h3 id="countdown_box">
                         <ul class="nav nav-list">
 
                             <li{% if request.path == '/' %} class="active"{% endif %}>
-                                <a href="{{ contest_url() }}">{{ _("Overview") }}</a>
+                                <a href="{{ contest_url() }}">{% trans %}Overview{% endtrans %}</a>
                             </li>
                             <li{% if request.path == '/communication' %} class="active"{% endif %}>
-                                <a href="{{ contest_url("communication") }}">{{ _("Communication") }}
-                                    <span id="unread_count" class="label label-warning no_unread">{{ _("%d unread") % 0 }}</span>
+                                <a href="{{ contest_url("communication") }}">{% trans %}Communication{% endtrans %}
+                                    <span id="unread_count" class="label label-warning no_unread"></span>
```

Comment:
Forgot something?

Reply:
Yes, I forgot to remove it earlier. Upon rendering, that message is invisible, because of the `no_unread` class. It is then JavaScript that updates its content and its class, loading the translated message from `div#translation_unread`. Hence this was useless.